reasoner/controller/RootConjunctionController.java

```diff
@@ -49,7 +50,9 @@
 
     @Override
     public void initialise() {
+        long start = PerfCounters.getNanos();
         planner().plan(conjunction, new HashSet<>());
+        processorContext().perfCounters().timePlanningMs.add((PerfCounters.getNanos() - start) / 1_000_000);
```

Comment:
as noted before, let's not do any division because it's going to be non-performant. Keep it in nanos